vector/src/main/res/values/strings.xml

```diff
@@ -2533,8 +2533,8 @@
     <string name="ftue_auth_use_case_option_two">Teams</string>
     <string name="ftue_auth_use_case_option_three">Communities</string>
     <!-- Note to translators: the %s is replaced by the content of ftue_auth_use_case_skip_partial -->
-    <string name="ftue_auth_use_case_skip">Not sure yet? %s</string>
-    <string name="ftue_auth_use_case_skip_partial">You can skip this question</string>
+    <string name="ftue_auth_use_case_skip">Not sure yet? You can %s</string>
+    <string name="ftue_auth_use_case_skip_partial">skip this question</string>
```

Comment:
Happy to see that your plan to put the new strings at a temporary location to avoid making Weblate handle them is useful here!

Reply:
same! it does mean there's a slight delay with when the translations can be done but it does save us doing multiple redundant translations